### ISSUE TITLE
Football tables: Non-clickable text is underlined

### ISSUE THREAD

**Issue by bryophyta** [guardian/frontend#25244]
The column headings on the tables are underlined, which makes them look clickable even though they are not. This can be considered [an accessibility issue](https://equalizedigital.com/accessibility-checker/underlined-text/).

![Image](https://user-images.githubusercontent.com/37048459/179444717-3af125c5-779d-41c7-9474-95bbca7fc0f6.png)

I think this is one where we'd need to hear @HarryFischer's input on alternative styling before resolving.

**Comment by HarryFischer** [comment]
Please just remove underline styling - bold is enough @bryophyta

**Comment by bryophyta** [comment]
On further investigation, this styling is actually being set by the user agent stylesheet in Chrome and Firefox, although it looks like this default styling can [vary between browsers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr#default_styling). The styling targets `abbr[title]` elements, and it signals that (at least in some browsers) the title (which should contain a definition of the abbreviation) can be viewed by hovering over the text.

So the original issue about underlines for elements which are not links is not quite right: there is a valid, and seemingly standardised, reason for adding the underline here, and the markup is consistent with the w3 spec. However, we are unclear how often this functionality is used -- just from looking at the table, both me and some other developers failed to realise that you could hover over the headings to find definitions.

Having some definition here seems valuable, but perhaps it would be more discoverable to users if it was contained in a separate 'key' table (possibly converting the abbreviations into in-page links to the key?)

Actions:
1. Raise this with DAC to find out more.
2. Check how the current markup works for screen-readers
3. (Potentially) design a key table -- depending how 1 and 2 go.
4. Potentially add a new rule to our reset styles, given that browser defaults seem to be variable?

**Comment by SiAdcock** [comment]
It would be useful to compare the design of our football tables with other sites such as [the BBC](https://www.bbc.co.uk/sport/football/tables), [Sky Sports](https://www.skysports.com/premier-league-table) and [the FA's England Football](https://www.englandfootball.com/mens-senior-team/fixtures-results) site. Also would be a good idea to bring designers into this discussion. I suspect there hasn't been many updates to our table design for a long time 🙂

**Comment by bryophyta** [comment]
Thanks for this Simon! @HarryFischer do you think that there's any appetite/scope for revisiting the football table design any time soon? It's not a blocker, because I think we can treat the abbreviation/underline issue on its own if we want to. But if you're thinking that we might do a broader re-design some time soon then it might make sense to hold off on the underline issue until after that alternative design's in place.

**Comment by HarryFischer** [comment]
I would like to rethink these but it's not going to be a priority for a while. Shall we catch up and chat about a short term fix? @bryophyta

**Comment by VDuczekW** [comment]
we have no clear way forward and believe the tables need to a redesign and to be addressed more broadly
